README.md

```diff
@@ -14,7 +14,6 @@ An [Ansible AWX](https://github.com/ansible/awx) operator for Kubernetes built w
 <!--ts-->
 * [AWX Operator](#awx-operator)
```

Comment:
Note: we are in the process of moving this readme into official docs in the /docs folder. Please go there to find additional sections during this interim move phase.